pyproject.toml

```
@@ -1,52 +1,106 @@
 [build-system]
-requires = ["setuptools>=70.2.0", "wheel", "setuptools_scm[toml]>=5.0"]
-build-backend = "setuptools.build_meta"
+requires = ["hatchling", "hatch-vcs"]
+build-backend = "hatchling.build"
 
 [project]
 name = "evmchains"
 description = "Packaged metadata on Ethereum Virtual Machine (EVM) chains"
-urls = {source = "https://github.com/ApeWorx/evmchains"}
 readme = "README.md"
-requires-python = ">=3.8"
-keywords = ["ethereum", "evm", "blockchain"]
-license = {file = "LICENSE"}
+requires-python = ">=3.10"
+license = { file = "LICENSE" }
+keywords = [
+    "ethereum",
+    "evm",
+    "blockchain",
+]
+authors = [
+    { name = "ApeWorX", email = "admin@apeworx.io" },
+]
 classifiers = [
     "Development Status :: 5 - Production/Stable",
     "Intended Audience :: Developers",
-    "Intended Audience :: End Users/Desktop",
     "License :: OSI Approved :: MIT License",
     "Natural Language :: English",
     "Operating System :: MacOS",
     "Operating System :: POSIX",
     "Operating System :: Microsoft :: Windows",
     "Programming Language :: Python :: 3",
-    "Programming Language :: Python :: 3.8",
-    "Programming Language :: Python :: 3.9",
     "Programming Language :: Python :: 3.10",
     "Programming Language :: Python :: 3.11",
     "Programming Language :: Python :: 3.12",
     "Programming Language :: Python :: 3.13",
+    "Programming Language :: Python :: 3.14",
 ]
 dependencies = [
     "pydantic>=2.5.3,<3",
 ]
 dynamic = ["version"]
 
+[project.urls]
+Homepage = "https://github.com/ApeWorx/evmchains"
+Documentation = "https://github.com/ApeWorx/evmchains#readme"
+Issues = "https://github.com/ApeWorx/evmchains/issues"
+Source = "https://github.com/ApeWorx/evmchains"
+
 [project.optional-dependencies]
```

Please always use this
```suggestion
[dependency-groups]
```